bus-mapping/src/circuit_input_builder.rs

```diff
@@ -203,10 +203,12 @@ impl<'a> CircuitInputBuilder {
 }
 
 /// Retrieve the init_code from memory for {CREATE, CREATE2}
-pub fn get_create_init_code(step: &GethExecStep) -> Result<&[u8], Error> {
+pub fn get_create_init_code(step: &GethExecStep) -> Result<Vec<u8>, Error> {
```

Comment:
because the slice is borrowed from an `RefCell`, the borrowed sliced won't live out of the scope of `Ref<'_>`.
I remove the redundant `to_vec()` for caller.